### PR TITLE
Return code according to the number of outputted vulnerabilities

### LINKED ISSUE #41: Severity threshold
Klar currently allows to filter output by severity and has a threshold by high severity vulnerabilities count.

It would be nice to allow the customization of the threshold level, for example to include Medium vulnerabilities in the threshold count, and return 1 if any.

### PR DESCRIPTION
**Pull request by supereagle**
Fix  #41

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -104,17 +104,19 @@ func main() {
 	c := clair.NewClair(clairAddr)
 	vs := c.Analyse(image)
 	groupBySeverity(vs)
-	highSevNumber := len(store["High"]) + len(store["Critical"]) + len(store["Defcon1"])
+	vsNumber := 0
 
 	if useJSONOutput {
 		iteratePriorities(clairOutput, func(sev string) {
+			vsNumber = vsNumber + len(store[sev])
```
Comment (hashmap) [inline main.go]: what do you think about `vsNumber +=  len(store[sev])` ?

File: main.go [hunk 2]
```diff
@@ -104,17 +104,19 @@ func main() {
...
 			output.Vulnerabilities[sev] = store[sev]
 		})
 		enc := json.NewEncoder(os.Stdout)
 		enc.Encode(output)
 	} else {
 		fmt.Printf("Found %d vulnerabilities\n", len(vs))
 		iteratePriorities(clairOutput, func(sev string) {
+			vsNumber = vsNumber + len(store[sev])
```
Comment (hashmap) [inline main.go]: same question

File: main.go (hunk 1 above)
Comment (supereagle) [inline main.go]: It is OK for me, will update this.

**Comment by supereagle** [comment]
@hashmap  Have updated, PTAL.

**Pull request merged**